ZupIT/beagle-web-angular
# Feat/refactor page view

Closes https://github.com/ZupIT/beagle-web-angular/issues/149

**- What I did**
Refactor page view according to the new contract. As we are not going to allow break changes for this version, we are keeping compatibility so the previous version should keep working (but a warning should be shown) and the new version should also work.

**- How I did it**
Create page indicator component and associate it as a default component, change page view interface to handle new contract and allow context to control the actual page.

**- How to verify it**
The playground project in `/#/cloud/f5d7548ac6984460a068c67e84628de7/deprecatedPageView.json?platform=angular` has an example of the new page view and the old page view, both should be working

**- Description for the changelog**

Refactor page view
